### PR TITLE
Blog Post: Meet the UF Ambassadors – Andre Alves

### PR DESCRIPTION
**Pull request by andre-a-alves**
I hope this is an ok introductory blog post - it covers how I got into quantum computing, how I got involved with UF, and what projects I am working on right now. The post is dated for tomorrow, because I'm not sure what release schedule we're trying to follow (nor am I sure how many edits we will need to make).

### CODE REVIEW

File: posts/_markdown/ambassador_alves_intro.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+During a conversation with [Sarah Kaiser](https://www.sckaiser.com/), she mentioned her theory that the [Volkswagen’s 2015 emissions cheating scandal](https://en.wikipedia.org/wiki/Volkswagen_emissions_scandal) was a catalytic event in popularizing quantum computing.
+Her very plausible idea is that soon after the scandal, Volkswagen bought a D-Wave quantum annealer, which garnered significant media attention and made other large corporations look into the technology themselves.
+At that point, coverage of quantum computing allegedly both increased and the sentiment within that coverage improved.
+
+Since I have little experience with web scraping and sentiment analysis, this seemed like a great project to tackle.
+It is a tremendous learning opportunity that can likely also lead to an academic paper.
+My idea is to create a database of all articles on the topic from Reuters and the Associated Press, and then perform sentiment analysis on those articles.
+Once the analysis is complete, I plan to analyze how reporting and sentiment have changed before and after the emissions scandal.
+If that is successful, I can then turn my analysis towards Twitter.
```
Comment (nathanshammah) [inline posts/_markdown/ambassador_alves_intro.md:116]:
```suggestion
```

File: posts/ambassador_alves_intro.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+                                <p>Once I started attending the weekly Mitiq meetings (every Friday at 17:00 UTC on <a href="http://discord.unitary.fund/">discord</a>) to understand what was going on, I was amazed at the open and welcoming community I found at Unitary Fund. Along with that sense of community, following along during the meetings helped me understand what were priority issues, and I decided to tackle creating decorators for PEC error correction. Decorators already existed for ZNE at the time, so this task was essentially making it so users could have the same or similar options for how to implement both techniques within their programs. In addition to that contribution, I made a few others that were mostly focused on organizing testing and improving documentation.</p>
```
Comment (nathanshammah) [inline posts/ambassador_alves_intro.html:88]: would be great to add a link to the PEC decorators
```suggestion
                                <p>Once I started attending the weekly Mitiq meetings (every Friday at 17:00 UTC on <a href="http://discord.unitary.fund/">discord</a>) to understand what was going on, I was amazed at the open and welcoming community I found at Unitary Fund. Along with that sense of community, following along during the meetings helped me understand what were priority issues, and I decided to tackle creating decorators for PEC, a quantum error mitigation technique available in Mitiq. Decorators already existed for ZNE at the time, so this task was essentially making it so users could have the same or similar options for how to implement both techniques within their programs. In addition to that contribution, I made a few others that were mostly focused on organizing testing and improving documentation.</p>
```

File: posts/_markdown/ambassador_alves_intro.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+So, I had an idea to improve their design by building a quantum-based comparator circuit where phototransistors can be used in such a way where when one electrical path opens, it shorts out the other path.
+I also want to use an FPGA to receive the signal, since that will be much faster than a general-purpose microcontroller.
+
+![Quantum-based Comparator Circuit](../../images/ambassador_alves_intro_circuit.png)
+
+Building this device is obviously a multi-step process, and I am currently at the circuit-proof-of-concept stage.
+At this point, I have designed and conventionally simulated the circuit.
+When I get back home in mid-February, I will order the necessary components, build it, and test it out.
+If the circuit works as I imagine, the next step will be to program an FPGA to work with the incoming signal.
+The final step will be to put it all on a small PCB that can interact with another electrical device, hopefully by USB.
```
[the remark this answers is not in the document]
Comment (andre-a-alves) [reply posts/_markdown/ambassador_alves_intro.md]: Change incorporated, and also made when PCB is referenced earlier in the blog post.

File: posts/_markdown/ambassador_alves_intro.md (hunk 1 above)
Comment (andre-a-alves) [reply posts/_markdown/ambassador_alves_intro.md]: Change incorporated

File: posts/ambassador_alves_intro.html (hunk 2 above)
Comment (andre-a-alves) [reply posts/ambassador_alves_intro.html]: Change incorporated, and I also added a link to PEC decorator as suggested

**Review by crazy4pi314 — APPROVED** [review]
LGTM! 🎉 Thanks for posting this, and we will get it circulated on socials tomorrow

**Pull request merged**